### ISSUE TITLE
Firefox Drawer's Contents Overlapping Its Own Trigger

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#40]
# Description
```
Please provide a short description of what you have encountered below.
```
On Firefox, it was detected that the drawer's content's links are blocking the main trigger. Hence, we should fix this UI problem.

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```
Drawer works on both Chome and Firefox browsers.

# Current Behavior
```
Please specify the current behavior of software below.
```
Firefox's drawer is broken.

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```
Click on the drawer trigger button on any pages. You'll be redirected to the page (usually "Getting Started") instead of opening the drawer.

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Fixed in 5a2e56e4abffd8df380b67cd37a9500aa313e5f3

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
